### ISSUE TITLE
Poor accessibility on some links of home page

### ISSUE THREAD

**Issue by autumnfound** [jakartaee/jakarta.ee#348]
In looking over the home page of the Jakarta.ee site for regression in theme upgrades, some accessibility violations were found. While this does not impact functionality, this can heavily impact visually impaired users. Some of the more impactful violations include:

- Color contrast between foreground text and background are poor
- Broken page content skip link
- Empty links with no alternate text 

This bug will be used as an investigation point into jakarta.ee, and the solstice assets and theme behind the site. The aim is to improve accessibility and have pages with no major issues when possible.

Some tools used in checking page:
https://webaim.org/resources/contrastchecker/
https://wave.webaim.org/
https://wave.webaim.org/extension/
Note that the browser extensions are more reliable for results than the web based check.

**Comment by autumnfound** [comment]
Examples of color contrast issue:
Buttons vs button text color (ratio of 1.98:1 vs suggested 3:1):
![image](https://user-images.githubusercontent.com/23018241/60830682-c20f0f80-a185-11e9-99f4-f2e62c58de05.png)

RSS footer links (ration of 1.62:1 vs suggested 4.5:1):
![image](https://user-images.githubusercontent.com/23018241/60830919-48c3ec80-a186-11e9-9662-d79e303213ae.png)

**Comment by autumnfound** [comment]
@stephanieswart I've added you to the issue as there will have to be some discussions on what we will fix, and how. Let me know when you have some time and we can go over some potential fixes for the graphical design portion!

**Comment by autumnfound** [comment]
From the discussion Stephanie and I had, the following updates were decided on:

1. Update the button font to be at least 14px  
2. Update font-weight to be 400
3. Update the RSS footer links to be a dark grey rather than light

While this doesn't move the page to compliance, it moves it closer and makes the page more accessible. In terms of future content, the tools mentioned in the above comment will be used to be more mindful of users who are visually impaired.

**Comment by autumnfound** [comment]
Changes made in solstice assets for overall CSS changes:
https://github.com/EclipseFdn/solstice-assets/pull/87

**Comment by autumnfound** [comment]
This has been fixed and is now live
